combat-trainer.lic

```diff
@@ -1400,7 +1405,7 @@ class SpellProcess
                      .select { |skill| Time.now > (@training_spells[skill]['cyclic'] ? @training_cyclic_timer : @training_cast_timer) }
                      .select { |skill| @training_spells[skill]['night'] ? UserVars.sun['night'] : true }
                      .select { |skill| @training_spells[skill]['day'] ? UserVars.sun['day'] : true }
-    needs_training = game_state.sort_by_rate_then_rank(needs_training).first
+                     .sort_by { |skill| DRSkill.getxp(skill) }.first
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept